README.md

```diff
@@ -32,9 +33,19 @@ Download the [container-diff-windows-amd64.exe](https://storage.googleapis.com/c
 
 ## Quickstart
 
-To use container-diff you need two Docker images (in the form of an ID, tarball, or URL from a repo).  Once you have those images you can run any of the following differs:
+To use container-diff to perform single image analysis, you need one Docker image (in the form of an ID, tarball, or URL from a repo).  Once you have that images you can run any of the following analyzers:
```

Comment:
nit: Once you have that, you can run (notice inserted comma and removed "images")